datacenter/install/azure.md

```diff
@@ -0,0 +1,254 @@
+
+    ![The resource group blade of a Docker EE for Azure deployment](../images/ddc-azure-resource-group-blade.png)
+
+## What's in the new resource group?
+
+Everything you need to run Docker EE is deployed, and the resource group
+collects it all in one place. The ARM template defines the architecture
+as shown in the following diagram.
+
+![Architecture diagram of a Docker EE for Azure deployment](../images/ddc-azure-architecture.png)
+
+Here are some of the resources that are included in the resource group.
+
+-   **Public IP addresses:** You connect to UCP and DTR by pointing your
+    browser at the corresponding load balancers. Also, you can SSH to
+    manager nodes.
+
+    ![IP addresses of a Docker EE for Azure deployment](../images/ddc-azure-load-balancers-ip-addys.png)
+
+-   **Node scale sets:** Virtual machine scale sets for the manager and worker
+    nodes. For more info, see [What are virtual machine scale sets in Azure?](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-overview)
+-   **Virtual network:** Click the virtual network resource, which has a name
+    similar to `ddcevalswarm-vnet`, to examine its architecture. In the left pane,
+    click **Diagram** to see the network diagram.
+-   **Storage:** The Docker EE deployment sets up Azure storage for your images.
+    The storage account has a name similar to `f4ybe46tzepm4dtr`.
+    The DTR user interface enables viewing details, on the Settings > Storage
+    page.
+
+## View the UCP and DTR interfaces
+
+Manage Docker EE by connecting your web browser to the UCP and the DTR
+load balancers.
+
+1.  In the left pane of the resource group blade, click **Deployments**.
+    The Deployments blade opens, showing your Docker EE for Azure deployment.
+    Click the deployment with a name similar to
+    `docker.dockerdatacenterdocker-datacenter-20170413151533`
+    to open the Deployment Details blade.
+
+    ![Output URLs of an Docker EE for Azure deployment](../images/ddc-azure-outputs.png)
+
+2.  In the **Output** section, copy the URL in `UCPLOGINURL` and
+    paste it into a web browser. The UCP login page opens. To login, use the
+    DDC username and password you entered in the Settings blade.
+
+3.  Copy the URL in `DTRLOGINURL` and paste it into a new tab in the
+    web browser. The DTR interface page opens.
+
+## Connect to a manager node with SSH
+
+In addition to managing Docker EE with the user interface, you can
+connect to a manager node by using SSH. You need the private key that
+corresponds with the public key that you provided when you created the Docker EE
+deployment. When you `ssh` to a manager node, login with the username `docker`,
+not the username you entered in the Settings blade, which is for UI.
+
+```bash
+$ ssh docker@ucplb-feykn7mj53x5q.westus.cloudapp.azure.com -p 50000 -i private-key.rsa
+Enter passphrase for key 'private-key.rsa':
+Welcome to Docker!
+swarm-manager000000:~$

```

Comment:
In this case the command is straightforward, so we don't need to show the result of the command. This will help the syntax highlighter behave properly.